Source/WebCore/dom/Element.cpp

```diff
@@ -3828,6 +3844,13 @@ AtomString Element::computeInheritedLanguage() const
     return document().contentLanguage();
 }
 
+AtomString Element::langForStyleResolution() const
```

Comment:
I'd call this 'effectiveLang' or similar.

Reply:
effectiveLangForStyleResolution? or just effectiveLang?

Reply:
Done.